spec/requests/api/reports_spec.rb

```diff
@@ -117,7 +161,7 @@
 
       expect do
         api_basic_authorize action_identifier(:reports, :run)
-        run_post "#{reports_url(report.id)}", :action => "run"
+        run_post reports_url(report.id).to_s, :action => "run"
```

Comment:
Was this a related change? If not perhaps it could be put into a different PR? It looks to me like neither interpolation nor `to_s` is needed - this should already be a string